src/System/LoadingFailedException.cs

```diff
@@ -1,14 +1,12 @@
 using System;
-using System.Runtime.Serialization;
 
-namespace SFML
+namespace SFML.System
 {
     ////////////////////////////////////////////////////////////
     /// <summary>
     /// Exception thrown by SFML whenever loading a resource fails
     /// </summary>
     ////////////////////////////////////////////////////////////
-    [Serializable]
```

Comment:
I don't think I have any issues with this change (Serializable doesn't exist in .NET Core, correct?)